src/components/interface/settings/SettingsDropdown.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, withRouter } from 'react-router-dom';
+
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import SettingsIcon from '@material-ui/icons/Settings';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import { auth } from '../../../firebase/firebase';
+import styles from '../../styles';
+
+class SettingsDropdown extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      anchorEl: undefined
+    };
+  }
+
+  handleClick = (event) => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleClose = () => {
+    this.setState({ anchorEl: undefined });
+  };
+
+  handleLogout = () => {
+    auth.signOut();
+  };
+
+  handleSettingsRedirect = (event) => {
```

Comment:
it seems as if we don't utilize this `handleSettingsRedirect` function, we can remove this right?

Reply:
Ah so this was a leftover function after a refactor. The dropdown here has 2 buttons: The settings button and the Logout function. The logout function doesn't require a redirect, it just needs to invalidate the `auth` object.

Any thoughts on if we should keep using `<Link to="/settings">` or use a handler function `onClick={handleSettingsRedirect}`?